daemon/logger/journald/read.go

```diff
@@ -175,7 +175,28 @@ func (s *journald) drainJournal(logWatcher *logger.LogWatcher, config logger.Rea
 
 	// Walk the journal from here forward until we run out of new entries.
 drain:
-	for {
+	for processed := 0; ; processed++ {
```

Comment:
I am not a "go" person, but does `processed` need to be declared first as an unsigned 64 bit integer?

Reply:
That particular syntax implicitly declares it as a generic integer.  It should have been unsigned; fixing it.